lib/bolt_server/transport_app.rb

```diff
@@ -84,40 +81,29 @@ def scrub_stack_trace(result)
       result
     end
 
-    def error_result(error)
```

Comment:
The biggest change in this PR is *not* making schema errors into "result" like objects and instead treating them as any other RequestError. This has run against the task/plan acceptance tests in orch and no issues have come up. Still investigating manually. Primary: `fertile-massuh.delivery.puppetlabs.net`